src/Integration/Helpers/VsShellUtils.cs

```diff
@@ -111,28 +111,29 @@ public static void ActivateSolutionExplorer(IServiceProvider serviceProvider)
             dte.ToolWindows.SolutionExplorer.Parent.Activate();
         }
 
-        public static IEnumerable<Property> EnumerateProjectProperties(Project project, string propertyName)
+        public static IEnumerable<Property> GetProjectProperties(Project project, string propertyName)
         {
             // Try to find the property on the project level properties (relevant for legacy web projects only)
             Property property = FindProperty(project.Properties, propertyName);
-            if (property == null)
+
+            if (property != null)
+            {
+                return new Property[] { property };
+            }
+
+            if (project.ConfigurationManager != null)
```

Comment:
The configuration manager can be null for some project types. I'm not sure why we haven't encountered this before (something may have changed in the platform).